src/ServicePulse.Host/vue/src/components/failedmessages/MessageView.vue

```diff
@@ -125,51 +143,65 @@ async function downloadHeadersAndBody() {
 }
 
 async function downloadBody() {
-  const response = await useFetchFromServiceControl("messages/" + failedMessage.value.message_id + "/body");
+  const response = await useFetchFromServiceControl("messages/" + failedMessage.value?.message_id + "/body");
```

Comment:
most things in this function require failedMessage.value != null, so can put this guard at the top and then remove all the null checks below